### PR TITLE
Add hacbs-jdk7-builder

### PR DESCRIPTION
**Pull request by dwalluck**

### CODE REVIEW

**Review by stuartwdouglas — CHANGES_REQUESTED** [review]
This also needs an entry in https://github.com/redhat-appstudio/jvm-build-service-builder-images/blob/main/image-config.yaml

**Comment by stuartwdouglas** [comment]
Merged, thanks. @dwalluck do you want to update the main repo to use this? You will need to update the builder images sha (I think in a few different places), and add a new entry for this image in the system config yaml.

There may also be a few places that assume JDK 1.8 is the lowest, so that may also need some fixes.

**Comment by dwalluck** [comment]
> Merged, thanks. @dwalluck do you want to update the main repo to use this? You will need to update the builder images sha (I think in a few different places), and add a new entry for this image in the system config yaml.
> 
> There may also be a few places that assume JDK 1.8 is the lowest, so that may also need some fixes.

I will look into it.

**Comment by dwalluck** [comment]
> There may also be a few places that assume JDK 1.8 is the lowest, so that may also need some fixes.

Yes, min JDK is set to 8 in a couple places. I assume in terms of priority that JDK 7 should come last, though.

Do you need to upload the image to the official quay repository before I create a PR for main?

File: .tekton/hacbs-jdk7-builder-pull-request.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    pipelinesascode.tekton.dev/on-target-branch: "[main]"
+    pipelinesascode.tekton.dev/max-keep-runs: "2"
+spec:
+  params:
+    - name: git-url
+      value: "{{repo_url}}"
+    - name: revision
+      value: "{{revision}}"
+    - name: output-image
+      value: "quay.io/redhat-appstudio/pull-request-builds:hacbs-jdk7-builder-{{revision}}"
```
Comment (stuartwdouglas) [inline .tekton/hacbs-jdk7-builder-pull-request.yaml:16]: @Michkov can you create this repo please? I don't seem to have permission.

File: .tekton/hacbs-jdk7-builder-push.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    pipelinesascode.tekton.dev/on-target-branch: "[main]"
+    pipelinesascode.tekton.dev/max-keep-runs: "2"
+spec:
+  params:
+    - name: git-url
+      value: "{{repo_url}}"
+    - name: revision
+      value: "{{revision}}"
+    - name: output-image
+      value: 'quay.io/redhat-appstudio/hacbs-jdk7-builder:{{revision}}'
```
Comment (stuartwdouglas) [inline .tekton/hacbs-jdk7-builder-push.yaml:16]: @Michkov oops, I mean this repo.

File: .tekton/hacbs-jdk7-builder-pull-request.yaml (hunk 1 above)
Comment (stuartwdouglas) [reply .tekton/hacbs-jdk7-builder-pull-request.yaml:16]: oops, I actually mean the repo below, not this one.

**Pull request merged**